esmf-semantic-aspect-meta-model/src/test/resources/samm_2_1_0/characteristic-shape/org.eclipse.esmf.test/1.0.0/TestCharacteristicInstanceWithEntityDataType.ttl

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
+#
+# See the AUTHORS file(s) distributed with this work for additional
+# information regarding authorship.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+@prefix : <urn:samm:org.eclipse.esmf.samm.test:1.0.0#> .
+@prefix samm: <urn:samm:org.eclipse.esmf.samm:meta-model:2.1.0#> .
+@prefix samm-c: <urn:samm:org.eclipse.esmf.samm:characteristic:2.1.0#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+:TestCharacteristicInstanceWithEntityDataType a samm:Characteristic ;
```

Comment:
I think the model name is misleading: It does not actually use a Characteristic Instance with an Entity data type, since the Characteristic Instance uses `xsd:integer` data type. But I'm not sure whether it should be renamed (assuming the content is correct) or have its content changed (assuming the model name is correct).

Reply:
Would this minimal model be enough (the model name is correct):

:TestCharacteristicInstanceWithEntityDataType a samm:Characteristic ;
   samm:dataType :TestEntityDataType .

:TestEntityDataType a samm:Entity ;
   samm:preferredName "Test Entity"@en ;
   samm:description "A test Entity"@en ;
   samm:properties ( ) .